tests/test_photo.py

```diff
@@ -195,6 +195,21 @@ def test_send_url_gif_file(self, bot, chat_id):
         assert isinstance(photo.file_id, str)
         assert photo.file_id != ''
 
+    @flaky(3, 1)
+    @pytest.mark.timeout(10)
+    def test_send_file_unicode_filename(self, bot, chat_id):
+        """
+        Regression test for https://github.com/python-telegram-bot/python-telegram-bot/issues/1202
+        """
+        with open('tests/data/测试.png', 'rb') as f:
```

Comment:
this needs a `u'filename'` prefix for py2.7 to work

Reply:
Ah great.. that was what I was missing. Thanks :D